src/rep_tests/simple_kv/simple_kv.client.h

```diff
@@ -48,8 +48,14 @@ class simple_kv_client
     
     // from requests to partition index
     // PLEASE DO RE-DEFINE THEM IN A SUB CLASS!!!
-    virtual int get_partition_index(const std::string& key) { return 0; }
-    virtual int get_partition_index(const ::dsn::replication::test::kv_pair& key) { return 0; }
+    virtual int get_partition_index(int partition_count)
```

Comment:
why I can get partition_index by only knowing partition_count? I think the original std::string&key is still required.